### PR TITLE
Add visualisation of slowdown for Sampling with Frame track comparison

### PR DESCRIPTION
**Pull request by akopich**
Test: Unit, Manual
Bug: http://b/238307352

### CODE REVIEW

**Review by ronaldfw — APPROVED** [review]
LGTM, though would like to clarify the llvm header include

File: src/MizarModels/SamplingWithFrameTrackReportModelTest.cpp [hunk 1]
```diff
@@ -182,28 +182,42 @@ TEST_F(SamplingWithFrameTrackReportModelTest, DisplayedDataIsCorrect) {
     const uint64_t expected_baseline_count = kSfidToBaselineCounts.at(sfid);
     const uint64_t expected_comparison_count = kSfidToComparisonCounts.at(sfid);
 
+    constexpr double kNsInUs = 1'000;
```
[the remark this answers is not in the document]
Comment (akopich) [reply src/MizarModels/SamplingWithFrameTrackReportModelTest.cpp:185]: Let's get it back.

[the remark this answers is not in the document]
Comment (akopich) [reply src/MizarModels/SamplingWithFrameTrackReportModelTest.cpp:185]: Let's get it back.

File: src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h [hunk 2]
```diff
@@ -6,6 +6,7 @@
 #define MIZAR_WIDGETS_SAMPLING_WITH_FRAME_TRACK_REPORT_MODEL_H_
 
 #include <absl/container/flat_hash_map.h>
+#include <llvm/Support/Format.h>
```
[the remark this answers is not in the document]
Comment (akopich) [reply src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h]: Sorry, that's not needed. Thanks. Auto-import, I guess.

**Pull request merged**